core/tests/.browserstack.env.example

```diff
@@ -0,0 +1,11 @@
+// This is the sample environment file which is
+// required to run e2e tests on Browserstack.
+// To run the tests, create a new file,
+// .browserstack.env, copy the contents of
+// this file in it and then put in the
+// actual browserstack credentials.
```

Comment:
How is this going to be managed (securely)? Who has access to the credentials, who can actually run the browserstack tests, etc.? Can the credentials be different per-user?